spec/controllers/home_controller_spec.rb

```diff
@@ -15,9 +15,14 @@
           existing_disclosure_check.disclosure_report.completed!
         end
 
-        it "redirects to /steps/check/kind" do
-          get :index, session: { disclosure_check_id: existing_disclosure_check.id }
-          expect(response).to redirect_to("/steps/check/kind")
+        # it "redirects to /steps/check/kind" do
+        #   get :index, session: { disclosure_check_id: existing_disclosure_check.id }
+        #   expect(response).to redirect_to("/steps/check/kind")
+        # end
+
```

Comment:
Hey Andrew; are we ok to leave this in a comment?

Reply:
I'm leaving the comments in as the disclaimer is a temporary measure.